packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -3072,6 +3072,41 @@ void main() {
     expect(tester.testTextInput.setClientArgs!['readOnly'], isFalse);
   });
 
+  testWidgets('Sends "updateConfig" when obscureText is flipped', (WidgetTester tester) async {
+    bool obscureText = true;
+    late StateSetter setState;
+    final TextEditingController controller = TextEditingController(text: 'Lorem');
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: StatefulBuilder(
+            builder: (BuildContext context, StateSetter stateSetter) {
+          setState = stateSetter;
+          return EditableText(
+            obscureText: obscureText,
+            controller: controller,
+            backgroundCursorColor: Colors.grey,
+            focusNode: focusNode,
+            style: textStyle,
+            cursorColor: cursorColor,
+          );
+        }),
```

Comment:
The indentation is strange here, I think this is right:

```suggestion
        home: StatefulBuilder(
          builder: (BuildContext context, StateSetter stateSetter) {
            setState = stateSetter;
            return EditableText(
              obscureText: obscureText,
              controller: controller,
              backgroundCursorColor: Colors.grey,
              focusNode: focusNode,
              style: textStyle,
              cursorColor: cursorColor,
            );
          },
        ),
```

Reply:
Thx, you are right.